linux/keyman-config/keyman_config/convertico.py

```diff
@@ -4,42 +4,102 @@
 import numpy as np
 import os
 import sys
+import struct
 from PIL import Image
-
+Image.LOAD_TRUNCATED_IMAGES = True
 
 def changeblacktowhite(im):
-    data = np.array(im)   # "data" is a height x width x 4 numpy array
-    red, green, blue, alpha = data.T # Temporarily unpack the bands for readability
+	data = np.array(im)   # "data" is a height x width x 4 numpy array
```

Comment:
Whitespace appears to have been converted to tabs; can you return to spaces please?